web-bluetooth/manufacturer-data-filter-async-await.js

```diff
@@ -0,0 +1,49 @@
+async function onButtonClick() {
+  let manufacturerData = {};
```

Comment:
nit: Const, here and for most other variables in the example.

Reply:
I've replaced `let` with `const` when appropriate in other parts of this example. 
This one was not replaced as I wanted to convey this variable would be updated later.